doc/extending.md

```diff
@@ -61,3 +61,28 @@ can run the templating manually:
 ```
 ./baselayer/tools/fill_conf_values.py --config="config.yaml" static/js/component.jsx.template
 ```
+
+## Adding extrernal services
+
+If you want to add external services to your application,
+you can do so by adding them to the `config.yaml` file under the
+`services.external` key. Each service should have a unique name and a set of parameters that define how to connect to it. For example:
```

Comment:
I don't think there are any parameters about how to "connect" to it, the wording could be a bit confusing given that there are actually networking concerns that are separate from that. I would rephrase it to make it clear that the config here let's baselayer pull the external service from a separate github repo. Also would be good to mention the versioning (branch+sha or version), which is omitted at the moment